Project.toml

```diff
@@ -16,14 +17,14 @@ Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 WeakRefStrings = "ea10d353-3f73-51f8-a26c-33c1cb351aa5"
 
 [compat]
-DataValues = "≥ 0.4.6"
-IteratorInterfaceExtensions = "^0.1, ^1"
-OnlineStatsBase = "≥ 0.9.1"
-PooledArrays = "≥ 0.4.1"
-StructArrays = "≥ 0.3.2"
-TableTraits = "^0.3, ^0.4, ^1"
-WeakRefStrings = "≥ 0.5.7"
-Tables = "≥ 0.2.4"
+DataValues = "0.4.6"
+IteratorInterfaceExtensions = "0.1.1, 1"
```

Comment:
I don’t think this needs to be bumped to 0.1.1, the previous 0.1 should be fine.

Reply:
Ah, I just copied what was in DataFrames